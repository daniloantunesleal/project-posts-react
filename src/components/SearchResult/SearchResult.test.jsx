import { render, screen } from "@testing-library/react"
import { SearchResult } from "."

describe("<SearchResult />", () => {
    it("should have message of results found", () => {
        render(<SearchResult result={101} />)

        expect(screen.getByText("Foi encontrado 101 resultados")).toBeInTheDocument()
    })

    it("should match snapshot", () => {
        const { container } = render(<SearchResult result={101} />)

        // eslint-disable-next-line testing-library/no-node-access
        expect(container.firstChild).toMatchSnapshot()
    })
})