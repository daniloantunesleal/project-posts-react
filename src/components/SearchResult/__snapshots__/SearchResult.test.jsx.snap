// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchResult /> should match snapshot 1`] = `
<p>
  Foi encontrado 
  101
   resultados
</p>
`;
